storage_service/locations/models/dspace.py

```diff
@@ -121,6 +128,58 @@ def _get_metadata(self, input_path, aip_uuid):
         os.remove(mets_path)
         return kwargs
 
+    def _archive(self, src, dst):
+        """
+        Run the archiver in order to create the archive file based on the
+        preferred format. `dst` is the path of the archive file. The file
+        extension must not be included, instead this function will return
+        the final destination with the extension on it according to the
+        archive format preferred.
```

Comment:
Tip: [PEP257](https://www.python.org/dev/peps/pep-0257/) for docstrings suggests having a commit-message-like docstring: single line summary, optionally followed by a multi-line explanation.